src/cli.ts

```diff
@@ -120,3 +124,19 @@ function fixCommandProcess() {
     const cmd = `${eslintCmd} src test --ext .ts --ext .tsx --fix --config ${eslintConfig}`;
     childprocess.execSync(cmd, {stdio: 'inherit'});
 }
+
+
+/**
+ * Generates typedoc
+ */
+function generateTypedoc() {
+    const config = getProjectConfig();
+    const [typedocCmd, themeDir, outDir, tsConfig] = [
+        './node_modules/.bin/typedoc',
+        './node_modules/openfin-service-tooling/typedoc-template',
+        './dist/docs',
+        './src/client/tsconfig.json'
+    ].map(filePath => path.resolve(filePath));
+    const cmd = `${typedocCmd} --name "OpenFin ${config.SERVICE_NAME}" --theme ${themeDir} --out ${outDir} --excludeNotExported --excludePrivate --excludeProtected --hideGenerator --tsconfig ${tsConfig} --readme none`; // eslint-disable-line
```

Comment:
How will this work with layouts, which has the output path ./dist/docs/api?

Reply:
So I've spoke with phil - the "standard" should be `./dist/docs/api` across all projects.  I've updated this to match